Makefile

```diff
@@ -124,6 +128,31 @@ e2e-helm-deploy:
 		./.staging/helm/linux-amd64/helm install manifest_staging/charts/gatekeeper --name-template=gatekeeper --set image.repository=${HELM_REPO} --set image.release=${HELM_RELEASE} --set emitAdmissionEvents=true --set emitAuditEvents=true;\
 	fi;
 
+e2e-helm-upgrade-init: e2e-helm-install
+	@if [ $$(echo ${HELM_VERSION} | head -c 1) = "2" ]; then\
+		kubectl create clusterrolebinding tiller-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default;\
+		./.staging/helm/linux-amd64/helm init --wait --history-max=5;\
+		kubectl -n kube-system wait --for=condition=Ready pod -l name=tiller --timeout=300s;\
+		./.staging/helm/linux-amd64/helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts;\
+		./.staging/helm/linux-amd64/helm install gatekeeper/gatekeeper \
+			--version ${BASE_RELEASE} --name gatekeeper \
+			--set emitAdmissionEvents=true --set emitAuditEvents=true --wait;\
+	else\
+		./.staging/helm/linux-amd64/helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts;\
+		./.staging/helm/linux-amd64/helm install gatekeeper/gatekeeper \
+			--version ${BASE_RELEASE} --name-template gatekeeper \
+			--set emitAdmissionEvents=true --set emitAuditEvents=true --wait;\
+	fi;
+
+e2e-helm-upgrade:
+	@if [ $$(echo ${HELM_VERSION} | head -c 1) = "2" ]; then\
```

Comment:
Are the two branches of this if-statement identical?

Reply:
fixed